src/components/GitOpsListPage.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import Helmet from 'react-helmet';
+import { useTranslation } from 'react-i18next';
+import { useK8sWatchResource, ListPageHeader, ListPageBody } from '@openshift-console/dynamic-plugin-sdk';
+
+import { fetchAllAppGroups, getManifestURLs } from './utils/gitops-utils';
+// import { fetchAppGroups, /* fetchAllAppGroups, getManifestURLs  getPipelinesBaseURI */ } from './utils/gitops-utils';
+import { GitOpsAppGroupData } from './utils//gitops-types';
+import GitOpsList from './list/GitOpsList';
+
+// import { PageHeading /*, LoadingBox */ } from '@console/internal/components/utils';
```

Comment:
If leaving in and commenting out the code that we need to migrate or port helps with the port work, then I'm ok with it.  Just a not to make sure we are working with the latest and greatest from the static plugin and not from these commented out code.